concrete/src/Area/Layout/Column.php

```diff
@@ -177,6 +177,16 @@ protected function getSubAreaMaximumColumns()
         return 0;
     }
 
+    public function getSubAreaObject()
+    {
+        $layout = $this->getAreaLayoutObject();
```

Comment:
@aembler, I needed a check for `$layout` in my version of layout caching. Built that 2-3 years ago, can't remember why.
```php
if (!is_object($layout)) {
    return;
}
```